### PR TITLE
Sprint game

### PR DESCRIPTION
**Pull request by nata-kostina**
1. Implement Sprint game
2. Add links to Sprint game in Main Menu and on Textbook pages
3. Add ministatistics for the game
4. Display ministatisctics of each word on Textbook section for authorized users

### CODE REVIEW

File: src/data/constants.ts [hunk 1]
```diff
@@ -64,4 +64,19 @@ export const wordsPerPage = 20;
 export const wordsPerUnit = 600;
 export const pagesNum = 30;
 
-export const MIN_PASSWORD_LENGTH = 8;
\ No newline at end of file
+export const MIN_PASSWORD_LENGTH = 8;
```
Comment (RichardPickman) [inline src/data/constants.ts:67]: I would like to suggest to use namespaces for collections of variables that belongs sematically to defined entity. 
I assume that current file is not yet fullfilled.

File: src/pages/Games/SprintGame/GameStatistics.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    navigate(`${RouteNames.TEXTBOOK}`);
+  }
+
+  useEffect(() => () => {
+    if (shouldChangePhase) {
+      setPhaseAction(GamePhase.INIT);
+    }
+  });
+
+  useEffect(() => () => {
```
Comment (RichardPickman) [inline src/pages/Games/SprintGame/GameStatistics.tsx:47]: duplicate of UseEffect as i can see

File: src/pages/Games/SprintGame/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.card {
+  border: 1px solid #000000;
+  padding: 1rem;
+}
+
+.card__word {
```
Comment (RichardPickman) [inline src/pages/Games/SprintGame/styles.module.scss:6]: Empty style block

File: src/pages/Homepage/index.tsx [hunk 4]
```diff
@@ -4,7 +4,9 @@ import Layout from '../../components/Layout';
 const Homepage = () => {
   return (
     <Layout>
-      <p>Homepage</p>
+      <>
```
Comment (RichardPickman) [inline src/pages/Homepage/index.tsx:7]: Is those empty parentheses is necessary?

**Review by RichardPickman — APPROVED** [review]
Sweet!

File: src/pages/Homepage/index.tsx (hunk 4 above)
Comment (nata-kostina) [reply src/pages/Homepage/index.tsx:7]: It's a short syntax for React Fragment.

File: src/pages/Games/SprintGame/styles.module.scss (hunk 3 above)
Comment (nata-kostina) [reply src/pages/Games/SprintGame/styles.module.scss:6]: > Empty style block

As it was mentioned, styles will be written after functional implementation.

File: src/pages/Homepage/index.tsx (hunk 4 above)
Comment (nata-kostina) [reply src/pages/Homepage/index.tsx:7]: > Is those empty parentheses is necessary?

As it was mentioned, styles will be written after functional implementation.

File: src/pages/Games/SprintGame/GameStatistics.tsx (hunk 2 above)
Comment (nata-kostina) [reply src/pages/Games/SprintGame/GameStatistics.tsx]: > duplicate of UseEffect as i can see

Fixed.

File: src/data/constants.ts (hunk 1 above)
Comment (nata-kostina) [reply src/data/constants.ts:67]: > namespaces

Suggestion is added to the "To do" tasks list.

**Comment by nata-kostina** [comment]
5. Add statistics for learned words.

**Pull request merged**